src/App.tsx

```diff
@@ -517,7 +580,38 @@ function App() {
 
   return (
     <div className="d-flex py-3 bg-light" style={{minHeight: '100vh'}}>
+      <button
+        type="button"
+        className="btn btn-secondary"
+        onClick={() => setSettingsOpen(true)}
+        style={{position: 'absolute', top: 12, right: 12}}
+      >
+        <span className="fa fa-gear" />
+      </button>
       <RouterProvider router={router} />
+      <Modal isOpen={settingsOpen} toggle={() => setSettingsOpen(false)}>
+        <ModalHeader toggle={() => setSettingsOpen(false)}>Settings</ModalHeader>
+        <ModalBody>
+          <form>
+            <div className="form-group">
+              <label htmlFor="ordering">Player ordering</label>
+              <select
+                className="form-control"
+                id="ordering"
+                value={ordering}
+                onChange={e => setOrdering(e.target.value as PlayerOrdering)}
+              >
+                <option key="priority" value="priority">
```

Comment:
I notice you keep setting keys on option elements that are not in any sort of list – it has no effect!